.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,18 @@
+name: ci
```

Comment:
Perhaps we should rename this workflow to distinguish it from the `mkdocs.yml` file in the repo root.